wasmplugin/rules/coraza-demo.conf

```diff
@@ -41,11 +41,13 @@ SecRule REQUEST_HEADERS:Content-Type "^application/json" \
 # to the size of data, with files excluded. You want to keep that value as
 # low as practical.
 #
+# Running as a Wasm plugin, we expect Limit equal to MemoryLimit: it would be prevented buffering request body to files anyways.
+
 SecRequestBodyLimit 13107200
 
-SecRequestBodyInMemoryLimit 131072
+SecRequestBodyInMemoryLimit 13107200
 
-SecRequestBodyNoFilesLimit 131072
+# SecRequestBodyNoFilesLimit 131072
```

Comment:
I think `SecRequestBodyNoFilesLimit` is not implemented Coraza side, we should at least comment it out, not letting users think that it is enforced.